base/shared/src/main/scala/scalaz/ct/BifunctorFunctions.scala

```diff
@@ -0,0 +1,14 @@
+package scalaz
+package ct
+
+trait BifunctorFunctions {
+  @inline final def bimap[F[_, _], A, B, S, T](fab: F[A, B])(
+    as: A => S,
+    bt: B => T
+  )(implicit F: Bifunctor[F]): F[S, T] =
+    F.bimap(fab)(as, bt)
+  @inline final def lmap[F[_, _], A, B, S](fab: F[A, B])(as: A => S)(implicit F: Bifunctor[F]): F[S, B] =
```

Comment:
Another option is to call these `firstMap`, `secondMap` after `Strong` (first/second).